userspace/libsinsp/test/filter_compiler.ut.cpp

```diff
@@ -217,14 +225,16 @@ TEST(sinsp_filter_compiler, supported_operators)
 	test_filter_compile(factory, "c.true intersects ()");
 	test_filter_compile(factory, "c.true pmatch ()");
 	test_filter_compile(factory, "c.true in()");
+	test_filter_compile(factory, "c.buffer bcontains 12ab001fc5");
+	test_filter_compile(factory, "c.buffer bstartswith 48545450");
 
 	// operators incompatibilites
-	test_filter_compile(factory, "c.true bstartswith g", true);
-	test_filter_compile(factory, "c.true bstartswith 123Z", true);
-	test_filter_compile(factory, "c.true bstartswith abc_1", true);
-	test_filter_compile(factory, "c.true bstartswith g", true);
-	test_filter_compile(factory, "c.true bstartswith 123Z", true);
-	test_filter_compile(factory, "c.true bstartswith abc_1", true);
+	test_filter_compile(factory, "c.buffer bstartswith g", true);
```

Comment:
Thanks for this! Could you add a couple more testcases? No need to add all possible combination, just something that you feel we may break if we touch :)